lang_js/analyze/ast_js_build.ml

```diff
@@ -569,7 +578,7 @@ and expr env = function
        else A.YieldStar 
     in
     let e = expr_opt env eopt in
-    A.Apply (A.IdSpecial (special, tok), [e])
+    A.Apply (A.IdSpecial (special, tok), (e |> Common.opt_to_list))
```

Comment:
Do `None`s get flattened to `[]` by this?

Reply:
Yes.